pyiceberg/catalog/rest.py

```diff
@@ -156,8 +156,10 @@ class RegisterTableRequest(IcebergBaseModel):
 class TokenResponse(IcebergBaseModel):
     access_token: str = Field()
     token_type: str = Field()
-    expires_in: int = Field()
-    issued_token_type: str = Field()
+    expires_in: Optional[int] = None
+    issued_token_type: Optional[str] = None
+    refresh_token: Optional[str] = None
+    scope: Optional[str] = None
```

Comment:
This is a nit, could we use Field annotation like the attributes above?
[Example](https://github.com/apache/iceberg-python/blob/5209874cd4685427520b67b64188f5e9919f2816/pyiceberg/table/snapshots.py#L141)

```suggestion
    expires_in: Optional[int] = Field(default=None)
    issued_token_type: Optional[str] = Field(default=None)
    refresh_token: Optional[str] = Field(default=None)
    scope: Optional[str] = Field(default=None)
```

Reply:
Fixed. Thanks for the suggestion. A curious question: what do we prefer a Field annotation here?

Reply:
From my perspective, just styling - it makes it consistent with how we are labeling the other Models in the repository